src/styles/index.scss

```diff
@@ -1,3 +1,97 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body,
+html {
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: white;
+}
+
+.stopwatch {
+  width: 81vmin;
+  height: 81vmin;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch__outer-ring {
+  width: 100%;
+  height: 100%;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+}
+
+.stopwatch__inner-ring {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch__hand {
+  position: absolute;
+  bottom: 50%;
+  transform-origin: bottom;
+  transition: transform 0.5s ease-in-out;
+}
+
+.stopwatch__hand--minute {
+  width: 3vmin;
+  height: 20vmin;
+  background: #0700ff;
+  border-radius: 2px;
+  animation: rotate-minute 3600s steps(60) infinite;
+}
+
+.stopwatch__hand--second {
+  width: 1.5vmin;
+  height: 38vmin;
+  background: #2c8000;
+  border-radius: 2px;
+  animation: rotate-second 60s linear infinite;
+}
+
+.stopwatch__center-circle {
+  width: 5vmin;
+  height: 5vmin;
+  background: #f6a603;
+  border-radius: 50%;
+  position: absolute;
+  z-index: 10;
+}
+
+.stopwatch--speed-up .stopwatch__hand--minute {
+  animation-duration: 600s;
+}
+
+.stopwatch--speed-up .stopwatch__hand--second {
+  animation-duration: 10s;
+}
+
+@keyframes rotate-minute {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes rotate-second {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
```

Comment:
pls combine these 2 animations to 1, because they do the same thing and reuse where needed